spec/controllers/suscribir/suscripciones_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+module Suscribir
+  describe SuscripcionesController do
+    routes { Suscribir::Engine.routes }
+
+    describe "desuscribir" do
+      let(:suscripcion) { create(:suscripcion) }
+
+      it "debe borrar la suscripcion si el token es correcto" do
+        params_baja_por_email = { suscripcion_id: suscripcion.id, token: suscripcion.token }
+        delete :desuscribir, params_baja_por_email
+        expect(Suscribir::Suscripcion.where(id: suscripcion.id)).to be_empty
+      end
+
+      it "no debe borrar la suscripcion si el token es incorrecto" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.